snap-style.gemspec

```diff
@@ -20,7 +20,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rubocop", "~> 0.63.0"
+  spec.add_dependency "rubocop", "~> 0.81.0"
+  spec.add_dependency "rubocop-rails", "~> 2.5.2"
```

Comment:
These versions are the minimum required to support Ruby 2.7 and should maintain compatibility with 2.3.